Apps/phthreatgrid/threatgrid_connector.py

```diff
@@ -408,7 +408,8 @@ def _query_file(self, param):
             'osver': '',
             'source': '',
             'vm': param.get('vm', ''),
-            'playbook': param.get('playbook', 'default')
+            'playbook': param.get('playbook', 'default'),
+            'sample_password': param.get('file_password')
```

Comment:
Now that you've rename the param `file_password` to `sample_password`. 
Should this be updated too? 
(i. e. `'sample_password': param.get('sample_password')`)